source/set_up_3dsecure/index.html.md.erb

```diff
@@ -9,12 +9,14 @@ weight: 170
 
 Strong Customer Authentication (SCA) makes online payments more secure for users and reduces fraud. SCA is part of the Payment Service Directive 2 (PSD2).
 
-If you take payments from the European Economic Area (EEA), defined as the European Union, Norway, Iceland and Liechtenstein, you must implement SCA by 31 December 2020.
+If you take payments from the European Economic Area (EEA), defined as the European Union, Norway, Iceland and Liechtenstein, you must implement SCA when you [go live](/switching_to_live/#go-live).
 
 If you only take payments within the UK, you must implement SCA by 14 September 2021.
 
 To implement SCA, you must set up 3D Secure 2 (3DS2) or 3D Secure 1 (3DS1) payment authentication.
 
+Payment service providers (PSPs) recommend setting up 3DS2, because 3DS1 will be retired in 2022.
```

Comment:
My understanding is that Mastercard plans to “sunset” (Worldpay’s words) 3DS1 in October 2022 but I’m not sure about Visa and the like. So we might want to weaken this slightly and say that 3DS1 will begin to be retired in 2022. We could weave this in with the extra fees for 3DS1 (some of which are already in place) and say something like 3DS1 is becoming more expensive and will begin to be retired altogether in 2022.

Reply:
Ah thanks @alexbishop1. We don't say anything about 3DS1 fees at the moment - @stevenjmesser, should we add something on that?

Reply:
Fees are between the service and their PSP, but it would be helpful to add what we know about the bigger picture. Are these tech docs the best place for that though? Perhaps we should iterate the 3DS page on the product pages.

Reply:
Makes sense - shall we go with this as is in the tech docs in that case (with Alex's suggested tweak to "begin to be retired")?

Reply:
I reckon so, yeah.

Reply:
@stevenjmesser @alexbishop1 Tweaked in https://github.com/alphagov/pay-tech-docs/pull/541/commits/3630dacdeba3e4986220613f04acc88a372020ce and ready for a green tick if you're both happy